src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1035,7 +1035,7 @@ private uint getMethodAttribsInternal(MethodDesc method)
                 DefType owningDefType = method.OwningType as DefType;
                 if (owningDefType != null && VectorOfTFieldLayoutAlgorithm.IsVectorOfTType(owningDefType))
                 {
-                    throw new RequiresRuntimeJitException("This function is using SIMD intrinsics, their size is machine specific");
+                    VerifyMethodSignatureIsStable(method.Signature);
```

Comment:
This needs to check that the abi of owningDefType is stable in addition to the signature. I think that will fix the failures you are seeing in the X64 tests.

Reply:
Unfortunately it don't fixes Debug build problem.

Reply:
:-/ Well, It looks reasonable now, but I don't know why its failing then. Let us know when you find out why.